coralproject/talk
# [CORL-2673] update copy for rejected ancestor error

## What does this PR do?
Updates user facing copy for error that occurs when attempting to reply to a comment that has been rejected or whos ancestor has been rejected since the page was refreshed.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?
None.

## Does this PR introduce any new environment variables or feature flags? 
No.

## If any indexes were added, were they added to `INDEXES.md`?
n/a

-->

## How do I test this PR?
1. Create a comment (A) and a reply (B)
2. Open a permalink view to B
3. As a moderator, reject A
4. Without refreshing permalink view, submit reply to B
5. Verify that you see the message "A comment earlier in this conversation thread was removed. No additional replies can be submitted."
 
 
## How do we deploy this PR?
No special considerations should be required
